components/scg/src/scg-model-objects.js

```diff
@@ -392,28 +391,9 @@ SCg.ModelLink.prototype.getConnectionPos = function (from, dotPos) {
  */
 SCg.ModelLink.prototype.setContent = function (content, contentType) {
     this.content = content;
-    this.contentType = contentType ? contentType : 'string';
+    this.contentType = contentType;
```

Comment:
There was a default type (string) for links.
![image](https://github.com/ostis-ai/sc-web/assets/36566977/640ef3b1-e0c7-4f89-8232-c501512d232c)
![image](https://github.com/ostis-ai/sc-web/assets/36566977/781b4495-fecb-4c74-b4ba-a720159800e9)